Solutions/Palo Alto - XDR (Cortex)/Detection Queries/WildFire Malware Detection.yaml

```diff
@@ -0,0 +1,33 @@
+id: 961672e7-15db-4df1-9bab-dc4f032b9b6f
+name: Palo Alto WildFire Malware Detection Rule
+description: |
+  'The query checks for specifically WildFire Malware and returns the result with Entities and hosts involved and count of attakcs.'
+severity: High
+requiredDataConnectors:
+  - connectorId: CommonSecurityLog
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1562
+query: |
+  CommonSecurityLog
+  | where ingestion_time() > ago(1d)
```

Comment:
DeviceVendor and DeviceProduct both are missing from the query. This query will fire across all the CEF products.